src/types.rs

```diff
@@ -281,3 +281,29 @@ pub struct AttachmentAction {
     pub room_id: String,
     pub created: String,
 }
+
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase", default)]
```

Comment:
Ooh... that would've been nice to know about.

Reply:
Yeah found that one out today by using this to convert the expected result to rust struct https://transform.tools/json-to-rust-serde